### PR TITLE
[typescript][11.qnamaker] Fixed typos for README.md and .bot file

### PR DESCRIPTION
**Pull request by joseques**
## Proposed Changes
- Minor changes to typos the README.md
- Fixed invalid syntax in `.bot` file that breaks the sample
- Fixed invalid syntax in `.qna` file that breaks the sample

### CODE REVIEW

File: samples/javascript_typescript/11.qnamaker/README.md [hunk 1]
```diff
@@ -48,11 +48,9 @@ QnA Maker enables you to power a question and answer service from your semi-stru
 One of the basic requirements in writing your own Bot service is to seed it with questions and answers. In many cases, the questions and answers already exist in content like FAQ URLs/documents, product manuals, etc. With QnA Maker, users can query your application in a natural, conversational manner. QnA Maker uses machine learning to extract relevant question-answer pairs from your content. It also uses powerful matching and ranking algorithms to provide the best possible match between the user query and the questions.
 
 # Deploy this bot to Azure
-You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on.
+You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on. Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher.
```
Comment (VictorGrycuk) [inline samples/javascript_typescript/11.qnamaker/README.md:51]: Can you adapt the Deploy section to match the one used in [netcore](https://github.com/southworkscom/BotBuilder-Samples/blob/78a021e12898d684f8c48bb72c6c2ca332e62c99/samples/csharp_dotnetcore/13.basic-bot/README.md#deploy-this-bot-to-azure)?

**Review by VictorGrycuk — CHANGES_REQUESTED** [review]

File: samples/javascript_typescript/11.qnamaker/README.md (hunk 1 above)
Comment (joseques) [reply samples/javascript_typescript/11.qnamaker/README.md:51]: Solved @VictorGrycuk

**Review by dfavretto — CHANGES_REQUESTED** [review]
[line 29 in file sample.qna](https://github.com/southworkscom/BotBuilder-Samples/pull/82/files#diff-fe0ff63770b7a8d51aa6745ed793c45fL29) is missing a comma, just like the other one added in the same file

**Comment by joseques** [comment]
Corrected @dfavretto

**Review by VictorGrycuk — APPROVED** [review]
👍 +1

**Review by dfavretto — APPROVED** [review]

**Comment by dfavretto** [comment]
Closed due to Microsoft#702

**Pull request closed without merging**